k8s-ingress/ingress-controller.yaml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: l5d-system
```

Comment:
It was a bit confusing to me that this file creates all of its resources in a separate namespace. From running the command, it looked like it was going to create everything in the default namespace. Maybe we could add a comment in the README? Or just created all of these resources in the default namespace?